psydac/core/bsplines.py

```diff
@@ -556,7 +569,7 @@ def make_knots(breaks, degree, periodic, multiplicity=1, out=None):
 
     if periodic:
         assert len(breaks) > degree
-
+    breaks = np.asarray(breaks)
```

Comment:
As above

Reply:
Done in https://github.com/pyccel/psydac/commit/54fa1fb4c2afba79c77392308a193bad59161fee